.github/workflows/benchmarks.yml

```diff
@@ -193,4 +193,4 @@ jobs:
             .asv/results
             asv-cont-output.log  
             asv-compare-output.log
-            asv-compare-changed-output.log
+            asv-compare-changed-output.log
```

Comment:
Is there any change here? Do you want to unstage this file since the before and after diffs are identical?